server/src/Addon.cs

```diff
@@ -222,6 +240,12 @@ void OnGUIApplicationLauncherDestroyed ()
         /// </summary>
         public void OnDestroy ()
         {
+            GameEvents.onGamePause.Remove(OnGamePause);
+            GameEvents.onGameUnpause.Remove(OnGameUnpause);
+            core.OnServerStarted -= OnServerStarted;
```

Comment:
Is this the part that fixes the leak?

Reply:
yep.  `GameEvents.onGamePause` is static, so anything referenced from there will never be garbage collected.  And `core.OnServerStarted` is effectively static, because it's a member of the singleton Core object which will never be destroyed.